src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -73,6 +92,33 @@ export const ALL_VIEWS = deepFreeze([
  * displays with a custom render function.
  */
 export const SPECIAL_FIELDS = {
+  '% query': {
+    fields: [],
+    sortField: false,
+    renderFunc: () => {
+      return <Container>% query</Container>;
+    },
+  },
+  transaction: {
+    fields: ['project.name', 'transaction'],
+    sortField: 'transaction',
+    renderFunc: (data, {organization, location}) => {
+      const target = {
+        pathname: `/organizations/${organization.slug}/events/`,
+        query: {
+          ...location.query,
+          transactionSlug: `${data['project.name']}:${data.transaction}`,
+        },
+      };
+      return (
+        <Container>
+          <Link css={overflowEllipsis} to={target} data-test-id="event-title">
```

Comment:
As an aside, thoughts on having an `aria-label` to describe this link, and then using that as a selector instead of a `data-test-id` attribute?